x-pack/plugins/viz_editor/public/frame/main/main.tsx

```diff
@@ -17,6 +18,8 @@ import {
   EuiPageContent,
   EuiPageContentBody,
   EuiPageSideBar,
+  // @ts-ignore
+  EuiSuperSelect,
```

Comment:
I don't see you using SuperSelect in this file- also, I believe most current implementations are actually using EuiComboBox with single selection enabled.